managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+   - **Username**: The username to use in conjunction with the source control system.
+   - **Password**: The password to use in conjunction with the source control system.
+   - **Private key passphrase**: If the SSH private key used is protected by a passphrase, you can configure a key passphrase for the private key.
+   - **Private Key**: Copy or drag and drop the actual SSH Private Key to be used to authenticate the user to the source control system by using SSH.
+
+### Viewing the List of Workflows in a Repository
+
+When your repository is added, the repository shows in the *Repositories* list. You can now click the entry for the repository to open it so that you can scan for existing workflows.
+If multiple workflows are found, click the corresponding number to view the list of workflows, **3** in this case.
+
+   ![Repository workflows](../images/embedworkflow_workflownumbers.png){: caption="Repository workflows" caption-side="bottom"}
+
+Alternatively, click **Automation** > **Embedded Workflows**. Then, click **Workflows** to see the workflows list.
+
+### Refreshing Repositories
+
+{{ site.data.product.title_short }} allows you to refresh a targeted repository or all repositories in your inventory to ensure that your workflows are current.
+
+Refresh a targeted repository:
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Repositories**.
+
+2.  Click a repository.
+
+3.  Click **Configuration**, then ![Refresh this Repository](../images/2003.png) (**Refresh this Repository**).
+
+Alternately, you can refresh some or all repositories from the list view:
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Repositories**.
+
+2.  Check those repositories to refresh. Click **Check All** to select all repositories.
+
+3.  Click **Configuration**, then ![Refresh Selected Workflow Repositories](../images/2003.png)(**Refresh Selected Workflow Repositories**).
+
+### Adding Credentials
+
+Credentials are used by {{ site.data.product.title_short }} for authentication when running embedded workflows against machines, synchronizing with inventory sources, and importing project content from a version control system.
+
+{{ site.data.product.title_short }} can store credentials that are used by embedded workflows. Credentials saved in {{ site.data.product.title_short }} are matched and executed with a workflow when it runs.
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Credentials**.
+
+2.  Click **Configuration**, then ![Add New Credential](../images/1862.png) (**Add New Credential**).
+
+3.  Provide a **Name** for the credential.
+
+4.  Select the **Credential Type**. More fields appear depending on the type chosen.
+
+5.  Click **Add**.
+
+#### Credential Types
```

Comment:
Not all of the credentials below are used by Embedded Workflows.  All of them are Ansible however, so I'd be good if we kept that content for Embedded Ansible.

Workflows only has "SCM" and "Workflow" types.

Reply:
I have removed the types other than SCM (workflow needs to be added) as requested.